data.tf

```diff
@@ -5,7 +5,7 @@ data "aws_ami" "int_tableau_linux" {
     name = "name"
 
     values = [
-      "dq-tableau-linux-279*",
+      "dq-tableau-linux-285*",
```

Comment:
I assume the new image has a different Number?

Reply:
Thanks @abouhouly 285 is the new number.  It was 284 on the image that failed. 279 is the last 'safe' build.